src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -1717,6 +1718,18 @@ void testGetOfflinePlayerIfCached_playerRegistered()
 		assertEquals(playerMock, offlinePlayer);
 	}
 
+	@Test
+	void testGetTags_blockRegistry() {
+		Iterable<Tag<Material>> blockTags = server.getTags(Tag.REGISTRY_BLOCKS, Material.class);
+		assertTrue(blockTags.iterator().hasNext());
+	}
+
+	@Test
+	void testGetTags_itemRegistry() {
+		Iterable<Tag<Material>> itemTags = server.getTags(Tag.REGISTRY_ITEMS, Material.class);
+		assertTrue(itemTags.iterator().hasNext());
+	}
```

Comment:
```suggestion
	void testGetTags_blockRegistry() 
	{
		Iterable<Tag<Material>> blockTags = server.getTags(Tag.REGISTRY_BLOCKS, Material.class);
		assertTrue(blockTags.iterator().hasNext());
	}

	@Test
	void testGetTags_itemRegistry() 
	{
		Iterable<Tag<Material>> itemTags = server.getTags(Tag.REGISTRY_ITEMS, Material.class);
		assertTrue(itemTags.iterator().hasNext());
	}
```


To be consistent with the rest of the code.